funcbench/bench.go

```diff
@@ -79,19 +79,23 @@ func (b *Benchmarker) execBenchmark(pkgRoot string, commit plumbing.Hash) (out s
 	}
 
 	// TODO(bwplotka): Allow memprofiles.
+	// 'go test' flags: https://golang.org/cmd/go/#hdr-Testing_flags
 	extraArgs := []string{"-benchtime", b.benchTime.String()}
 	extraArgs = append(extraArgs, "-timeout", b.benchTimeout.String())
 	benchCmd := []string{"bash", "-c", strings.Join(append(append([]string{"cd", pkgRoot, "&&", "go", "test", "-run", "^$", "-bench", fmt.Sprintf("^%s$", b.benchFunc), "-benchmem"}, extraArgs...), "./..."), " ")}
 
-	b.logger.Println("Executing benchmark cmd:", benchCmd)
+	b.logger.Println("Executing benchmark command for", commit.String())
+	b.logger.Println(benchCmd)
```

Comment:
Why two lines? This is one event, so you can print once. Just add `\n` char between  in the message (: